ethereum/transaction_status_task.go

```diff
@@ -0,0 +1,156 @@
+package ethereum
+
+import (
+	"context"
+	"time"
+
+	"github.com/centrifuge/go-centrifuge/errors"
+	"github.com/centrifuge/go-centrifuge/identity"
+	"github.com/centrifuge/go-centrifuge/queue"
+	"github.com/centrifuge/go-centrifuge/transactions"
+	"github.com/centrifuge/gocelery"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+const (
+	TransactionStatusTaskName string = "TransactionMintingTask"
+	TransactionTxHashParam    string = "TxHashParam"
+	TransactionTenantIDParam  string = "Tenant ID"
+	TransactionStatusSuccess  uint64 = 1
+)
+
+type transactionStatusTask struct {
+	transactions.BaseTask
+	timeout time.Duration
+	//state
+	ethContextInitializer func(d time.Duration) (ctx context.Context, cancelFunc context.CancelFunc)
+
+	//task parameter
+	txHash   string
+	tenantID identity.CentID
```

Comment:
`accountID`

Reply:
please dont do these changes now, it would be done in a single pr by me..

Reply:
https://app.zenhub.com/workspaces/centrifuge-5ba350114b5806bc2be90978/issues/centrifuge/go-centrifuge/617